src/setupTests.js

```diff
@@ -1,4 +1,3 @@
-import 'jest-enzyme'
 import { configure } from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
```

Comment:
Ah nice, I didn't know that. I'll remove it 👍

Reply:
Hmm. Removed it, and removed the configuration from package.json, but tests then fail:

<img width="605" alt="Screenshot 2020-11-25 at 09 20 42" src="https://user-images.githubusercontent.com/7355199/100200897-936f1180-2eff-11eb-833c-664e43621cfe.png">

I think that that setup only applies to the tests run for the app-shell, not for the tests run for this app.

Reply:
Ah, a shame.. fair enough

Reply:
Yeah it’s included for the shell tests, but not for app tests (currently) - it might be worth doing, though, since I think enzyme has become fairly ubiquitous